src/styles/images/aspect-ratio-custom/index.njk

```diff
@@ -8,6 +8,6 @@ stylesheets:
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-l">Do you have a V11 reminder letter printed in your name?</h1>
-    <img src="v11.gif" alt="">
+    <img class="govuk-!-width-one-third" src="v11.svg" alt="">
```

Comment:
I noticed the SVG version of the file has a transparent background. I think in this case we would want the background in the SVG to be white, so that the image still functions properly when users are forcing a dark mode styling or using high-contrast settings.

Reply:
Good spot, thanks!